Source/WebInspectorUI/UserInterface/Base/ImageUtilities.js

```diff
@@ -126,14 +126,6 @@ WI.ImageUtilities = class ImageUtilities {
         });
         return image;
     }
-
-    static supportsCanvasPathDebugging()
-    {
-        return "getPath" in CanvasRenderingContext2D.prototype
-            && "setPath" in CanvasRenderingContext2D.prototype
-            && "currentX" in CanvasRenderingContext2D.prototype
-            && "currentY" in CanvasRenderingContext2D.prototype;
-    }
```

Comment:
Aside: under what circumstance would this have not returned false before? Should we ever have had an instance of Web Inspector where the Web Inspector Additions weren't enabled?

Reply:
i think WebKitLegacy didn't have this enabled